sympy/physics/vector/frame.py

```diff
@@ -790,8 +812,8 @@ def orient_body_fixed(self, parent, angles, rotation_order):
 
         >>> B1.orient_axis(N, N.x, q1)
         >>> B2.orient_axis(B1, B1.y, q2)
-        >>> B.orient_axis(B2, B2.x, q3)
-        >>> B.dcm(N)
```

Comment:
If this worked before this change, why doesn't it work now?

Reply:
Earlier we weren't checking loops , so it was working. It would work now , but would raise `loop warning`.

Reply:
Yes, I see it is a loop now. But maybe the case should be left in and show the warning here.

Reply:
I think we should add some message that doing this would raise a warning, but  I think it shouldn't raise a warning in docstrings.

Reply:
Currently there is nothing in the documentation that explains you'll get warnings for loops. If this showed a warning here, then users would be informed of that possibility.

Reply:
But if something raises warning then docstrings tests fail because of that warning. So how could we willingly raise warning in docstrings?

Reply:
Not sure, that would be a reason to leave it out.

Reply:
> docstrings tests fail because of that warning

we show tracebacks in the doctests. This will not cause the doctest to fail. The tester just looks to see if the input gives the reported output -- error or not. And it is liberal in interpretation, allowing you to use dots to exclude unnecessary clutter.

Reply:
Thanks Chris. If that works, I'd prefer that to removing this test.

Reply:
There are examples of Tracebacks in tensor.functions

Reply:
Thanks Chris, I would update that.

Reply:
@moorepants @smichr  I have been trying to catch warning in docstrings but haven't been successful and tracebacks, dots aren't working in case of warnings. I'm trying to find more ways, but if you know some other way then I would like to know that.

Reply:
One possible work around could be to add `Warns` section in all `orient()` docstring and avoid loops in docstrings, this way user could know about warnings and we could avoid them at the same time.

Reply:
>  One possible work around could be to add Warns section in all orient() docstring and avoid loops in docstrings, this way user could know about warnings and we could avoid them at the same time.

Fine also.